javascript/ql/lib/semmle/javascript/PackageExports.qll

```diff
@@ -120,14 +120,21 @@ private DataFlow::Node getAValueExportedByPackage() {
   exists(ImmediatelyInvokedFunctionExpr func, DataFlow::ParameterNode prev, int i |
     prev.getName() = "factory" and
     func.getParameter(i) = prev.getParameter() and
-    result = func.getInvocation().getArgument(i).flow().getAFunctionValue().getAReturn() and
-    DataFlow::globalVarRef("define").getACall().getArgument(1) = prev.getALocalUse() and
+    DataFlow::globalVarRef("define").getACall().getArgument(any(int a | a >= 1)) =
```

Comment:
can `int a` ever be larger than 2? and couldn't it also be `0`? like in 1.3.2 here: https://requirejs.org/docs/api.html#define Perhaps, this should simply be `getAnArgument`

Reply:
You seem to be right, it can be any argument.